app/services/services.py

```diff
@@ -430,3 +430,32 @@ def hash_password(password: str):
 
     # Hashing the password
     return  bcrypt.hashpw(pw_bytes, salt)
+
+
+def get_current_user(request: Request):
+    """
+    Parameters:
+        request: The request object.
+
+    Returns:
+        bool: The user session
+    """
+    return request.session
+
+
+def is_owner(request: Request, video_owner: str) -> bool:
+    """
+    Checks if the current user is the owner of a video.
+
+    Parameters:
+        request: The request object.
+        video_owner (str): The username of the video owner.
+
+    Returns:
+        bool: True if the current user is the owner; otherwise, False.
+    """
+    user = get_current_user(request)
+
+    if user:
+        return user['username'] == video_owner
+    return False
```

Comment:
get_current_user(request) return  a boolean
So how is it possible to do `user['username']`

Reply:
Return type has been updated in doctoring

Reply:
If user id a dict, you can refactor the code to this
```suggestion
        return user.get("username") == video_owner
```
@TolulopeJoel

Reply:
Fixed. Beautiful and simple, thanks.